ExtractData/PKHeX.ExtractData/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace PKHeX.ExtractData
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string folder = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
+            if (Path.GetFileName(folder).ToLower() == "debug")
```

Comment:
Doing these checks complicates things.  It looks like this program is expecting certain files with very specific names to be in very specific places.

I would recommend just checking for certain file or directory names in the working directory, or at the very least, provide documentation as to where the files should go.